### ISSUE TITLE
Add support for custom code.

### ISSUE THREAD

**Issue by aashutoshrathi** [yashLadha/yaas#1]
Something like this is used in Cowboy servers:
```sh
curl -i https://yaas.ml -F "url=fb.me/iiitvcc" -F "code=iiitvcc"
```

Not sure what is here for ngnix.

Right now response is:

```sh
HTTP/1.1 405 Not Allowed
Server: nginx/1.10.3
Date: Thu, 06 Dec 2018 11:33:23 GMT
Content-Type: text/html
Content-Length: 173
Connection: keep-alive

<html>
<head><title>405 Not Allowed</title></head>
<body bgcolor="white">
<center><h1>405 Not Allowed</h1></center>
<hr><center>nginx/1.10.3</center>
</body>
</html>
```

**Comment by yashLadha** [comment]
Yes, because if you monitor closely the `Actions.js` and particularly this [line](https://github.com/yashLadha/yaas/blob/afdb32a2ad2c5448e8af81fbb8a1ad5da048d47b/yaas-frontend/src/redux/Actions.js#L18). Them it is evident that the URL you are trying to use is not correct, Though the logic can be modified for handling this request but the length of the shortened code should be at least six characters because of the server block in nginx. @aashutoshrathi .
